draft-meunier-web-bot-auth-architecture.md

```diff
@@ -307,7 +307,7 @@ See {{signature-agent}} for more details.
 ## Use of TLS
 
 We reassess {{Section 7.1.2 of HTTP-MESSAGE-SIGNATURES}}.
-Clients SHOULD use TLS {{RFC5246}}
+Clients SHOULD use TLS {{RFC8446}}
```

Comment:
That's the right change. The `informative` section at the top should also be update to remove `RFC5246` and use `RFC8446` instead.

I'm not able to update your fork. Would be awesome if you do it.

Reply:
I will update it.

Reply:
Done.